Description: Restores the backward compatibility with maven-shared-utils 3.1
 This patch is required to build Maven 3.5.3 with Maven 3.5.2. It can be dropped
 once maven/3.5.3-1 is uploaded.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/src/main/java/org/apache/maven/shared/utils/logging/MessageBuilder.java
+++ b/src/main/java/org/apache/maven/shared/utils/logging/MessageBuilder.java
@@ -27,6 +27,27 @@
 public interface MessageBuilder
 {
     /**
+     * Append message content for DEBUG level.
+     * By default, bold cyan
+     */
+    @Deprecated
+    MessageBuilder debug( Object message );
+
+    /**
+     * Append message content for INFO level.
+     * By default, bold blue
+     */
+    @Deprecated
+    MessageBuilder info( Object message );
+
+    /**
+     * Append message content for ERROR level.
+     * By default, bold red
+     */
+    @Deprecated
+    MessageBuilder error( Object message );
+
+    /**
      * Append message content in success style.
      * By default, bold green
      */
--- a/src/main/java/org/apache/maven/shared/utils/logging/PlainMessageBuilder.java
+++ b/src/main/java/org/apache/maven/shared/utils/logging/PlainMessageBuilder.java
@@ -62,6 +62,24 @@
         return a( level ).toString();
     }
 
+    @Deprecated
+    public PlainMessageBuilder debug( Object message )
+    {
+        return a( message );
+    }
+
+    @Deprecated
+    public PlainMessageBuilder info( Object message )
+    {
+        return a( message );
+    }
+
+    @Deprecated
+    public PlainMessageBuilder error( Object message )
+    {
+        return a( message );
+    }
+
     public PlainMessageBuilder success( Object message )
     {
         return a( message );
--- a/src/main/java/org/apache/maven/shared/utils/logging/AnsiMessageBuilder.java
+++ b/src/main/java/org/apache/maven/shared/utils/logging/AnsiMessageBuilder.java
@@ -70,6 +70,27 @@
         return Style.ERROR.apply( ansi ).a( level ).reset().toString();
     }
 
+    @Deprecated
+    public AnsiMessageBuilder debug( Object message )
+    {
+        Style.DEBUG.apply( ansi ).a( message ).reset();
+        return this;
+    }
+
+    @Deprecated
+    public AnsiMessageBuilder info( Object message )
+    {
+        Style.INFO.apply( ansi ).a( message ).reset();
+        return this;
+    }
+
+    @Deprecated
+    public AnsiMessageBuilder error( Object message )
+    {
+        Style.ERROR.apply( ansi ).a( message ).reset();
+        return this;
+    }
+
     public AnsiMessageBuilder success( Object message )
     {
         Style.SUCCESS.apply( ansi ).a( message ).reset();
